lib/utils/indent-common.js

```diff
@@ -311,11 +311,9 @@ module.exports.defineVisitor = function create(
           token.type === 'HTMLEndTagOpen' ||
           token.type === 'HTMLComment')
     }
-    for (const token of tokenStore.getTokensBetween(
-      node.startTag,
-      endToken,
-      option
-    )) {
+    for (const token of endToken
+      ? tokenStore.getTokensBetween(node.startTag, endToken, option)
+      : tokenStore.getTokensAfter(node.startTag, option)) {
```

Comment:
I'd extract this into a separate variable `contentTokens` to make the for loop more readable. Also, please rename `option` to `options` while you're at it.

Reply:
Thank you for your review! I changed it.